src/ApiService/Tests/Integration/AzuriteStorage.cs

```diff
@@ -22,15 +22,11 @@ public Uri GetTableEndpoint(string _accountId)
     const string AccountName = "devstoreaccount1";
     const string AccountKey = "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==";
 
-    public Task<(string?, string?)> GetStorageAccountNameAndKey(string _accountId)
-        => Async.Task.FromResult<(string?, string?)>((AccountName, AccountKey));
+    public Task<(string, string)> GetStorageAccountNameAndKey(string accountId)
+        => Async.Task.FromResult((accountId, AccountKey));
```

Comment:
the test class here should ignore the `accountId`, it will always use the same account name

```suggestion
        => Async.Task.FromResult((AccountName, AccountKey));
```